app/views/Export/ExportTemplate.js

```diff
@@ -65,6 +51,7 @@ export const ExportTemplate = ({
   bodyLinkOnPress,
   ignoreModalStyling, // So first screen can be slightly different in tabs
 }) => {
+  useStatusBarEffect(lightTheme ? 'dark-content' : 'light-content');
```

Comment:
My understanding is that we are moving away from the in app Theme stuff.  should this just be
```suggestion
  useStatusBarEffect('dark-content');
```
?

Reply:
Ideally yeah. There are a few places in the app still using the legacy system, mostly because the buttons were built around theming.